### ISSUE TITLE
Use HTML tags for training HF model

### ISSUE THREAD

**Issue by EvilDrPurple** [Police-Data-Accessibility-Project/data-source-identification#29]
### Context

Part of #12 
We have a working Hugging Face model that predicts record type based on a website's URL. We want to update this to include a website's HTML tags as part of the training data.

### Requirements

* Hugging Face model should be trained with HTML tags and able to predict with high accuracy.

### Tests

* See if this improves model accuracy. Current model accuracy without HTML tags is ~60%

### Open questions

* Will using precision as a training metric over accuracy make the model more or less accurate?
* How accurate can the model get when training with 3 epochs or more rather than 1-2?
